composer.json

```diff
@@ -75,7 +76,7 @@
     },
     "extra": {
         "installer-paths": {
-            "web/core": ["type:drupal-core"],
+            "web/core": ["drupal/core"],
```

Comment:
I can't find the reason why we changed this on subs - other project templates seem to use `"web/core": ["type:drupal-core"]` as we had it so I think we should stick with this if it works